.github/workflows/cla-assistant.yaml

```diff
@@ -0,0 +1,47 @@
+name: "CLA Assistant"
+on:
+  issue_comment:
+    types: [created]
+  pull_request_target:
+    types: [opened, closed, synchronize]
+# explicitly configure permissions, in case your GITHUB_TOKEN workflow permissions are set to read-only in repository settings
+permissions:
+  actions: read
+  contents: read # this can be 'read' if the signatures are in remote repository
+  pull-requests: write
+jobs:
+  CLAAssistant:
+    runs-on: ubuntu-latest
+    steps:
+      - id: fetch-token
+        name: Fetch installation token
+        uses: actions/create-github-app-token@v1
+        with:
+          app-id: ${{ secrets.CLA_ASSISTANT_APP_ID }}
+          private-key: ${{ secrets.CLA_ASSISTANT_APP_PRIVATE_KEY }}
+          repositories: cla-signatures
+      - id: get-membership
+        name: "Determine gravitational membership"
+        env:
+          GITHUB_TOKEN: ${{ steps.fetch-token.outputs.token }}
+          USER: ${{ github.actor }}
+        continue-on-error: true
+        # Use the "Check Membership" API for user
+        # If the user is a member of the org expect a 204 HTTP Status Code
+        # If the above is success the process will exit 0
+        run: |
+          gh api "orgs/gravitational/members/${USER}"
```

Comment:
It might be worth checking if `state === "active"` in the response

Reply:
This call is actually the much simpler [API call](https://docs.github.com/en/rest/orgs/members?apiVersion=2022-11-28#check-organization-membership-for-a-user). It takes no parameters and returns `204 No Content`.

I can update it to use the other one. I just chose this one because it returns no data.

Reply:
Oh cool that's great, lets do this instead